packages/hub/config/custom-environment-variables.json

```diff
@@ -53,7 +53,13 @@
     "provisionerSecret": "PROVISIONER_SECRET"
   },
   "cardDrop": {
-    "sku": "CARD_DROP_SKU"
+    "sku": "CARD_DROP_SKU",
+    "email": {
+      "rateLimit": {
+        "count": {"__name": "HUB_EMAIL_CARD_DROP_RATE_LIMIT_COUNT", "__format": "number"},
```

Comment:
TIL this little syntax for typing config!

Reply:
ya… Javascript’s incredible laxness around typing makes this actually unnecessary but it just feels better to me to know these will come out as numbers without having to scatter `parseInt` everywhere they’re accessed 🤓